ui/src/containers/CreateClient.js

```diff
@@ -17,9 +31,29 @@ class CreateClient extends Component {
 
     constructor(props) {
         super(props)
-        this.baseState = this.state;
+        this.baseState = this.state
     }
 
+    componentDidUpdate() {
+      if (this.state.loading && !this.props.apps.isCreating) {
+        if (this.props.apps.createError) {
+          this.setState({
```

Comment:
I wanted to point out the usage of `setState` first but i think it's ok here because the component is a Wizard and needs to maintain internal state.

Reply:
That's a question if we it is better to change the code to have state completely in Redux.